tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -247,7 +247,7 @@ export class PodmanDesktopRunner {
   private setupPodmanDesktopCustomFolder(): object {
     const env: { [key: string]: string } = process.env as { [key: string]: string };
     const dir = join(this._customFolder);
-    console.log(`podman desktop custom config will be written to: ${dir}`);
+    console.log(`Podman Desktop custom config will be written to: ${dir}`);
```

Comment:
the two changes in this file are unrelated to this PR so I would make it part of another commit/PR

Reply:
ok.